partner_gender/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Partner Gender
+==============
+
+This module was written to extend the functionality of `base_contact`_ to
```

Comment:
I think the whole point is that *base_contact* is wrong. It makes many things that are totally independent.

I inherit the module just for [the view sheet of personal data](https://github.com/OCA/partner-contact/pull/118/files#diff-93b96a5c10e7af0bbc086441331b2017R27), but I agree that I don't need that dependency for anything else.

IMHO *base_contact* should be just that: a base for all modules that modify the contact form (BTW, it should be *base_partner* for more general usage). It should define the new views and nothing more. Then 3 modules out of there should appear:

- *base_contact_type*
- *base_contact_birthdate*
- *base_contact_nationality*

And, of course, *base_contact_gender*, which is what we are really talking about.